10fastfingershack/README.md

```diff
@@ -0,0 +1,2 @@
+#Requires Selenium and Chrome Webdriver
+Install selenium and specify location in the Python Script.
```

Comment:
Can you please add a link to 10fastfingers and describe a little about it.

Reply:
Done!